{"name": "Проверка наличия ошибки при создании пользователя который уже зарегистрирован", "status": "passed", "start": 1755162173562, "stop": 1755162173961, "uuid": "f1e30648-d700-47f1-8118-e14ae2cc7efc", "historyId": "12037716fbde77105fe41f460fa470ad", "testCaseId": "12037716fbde77105fe41f460fa470ad", "fullName": "tests.test_registration_user.TestRegistrationUser#test_registration_same_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_registration_user"}, {"name": "subSuite", "value": "TestRegistrationUser"}, {"name": "host", "value": "DESKTOP-27LVRII"}, {"name": "thread", "value": "14204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_registration_user"}], "titlePath": ["tests", "test_registration_user.py", "TestRegistrationUser"]}